src/openrct2/ride/Vehicle.cpp

```diff
@@ -8940,7 +8940,7 @@ void Vehicle::UpdateTrackMotionMiniGolfVehicle(Ride* curRide, rct_ride_entry* ri
 
     SetTrackType(tileElement->AsTrack()->GetTrackType());
     SetTrackDirection(direction);
-    var_CF = tileElement->AsTrack()->GetSeatRotation() << 1;
+    brake_speed = tileElement->AsTrack()->GetSeatRotation() << 1;
```

Comment:
![image](https://user-images.githubusercontent.com/1277401/145897177-982c6c49-3978-46a2-b022-dde42b857a7c.png)
Looks like this was wrong it should have been:
```
brake_speed = tileElement->AsTrack()->GetBrakeBoosterSpeed();
```